doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -43,34 +45,233 @@ On your computer, one of the following operating systems:
+            If you run the script as root, to fix permissions delete the installation folder and run the script again as a non-root user.
+
+      #. Run the following command in your terminal:
+
+         .. parsed-literal::
+            :class: highlight
+
+            /Users/*yourusername*/ncs-lcs/nrfutil toolchain-manager launch --shell --chdir "/Users/*yourusername*/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
+
+         This makes west and other development tools in the |NCS| toolchain environment available in the same shell session.
+
+         .. caution::
+            When working with west in the customer sampling release version of |NCS|, you must always use this shell window.
+
+      #. Run the following commands in your terminal to install the correct lxml dependency:
+
+         .. parsed-literal::
+            :class: highlight
+
+            pip uninstall -y lxml
+            pip install lxml
+
+      #. Install the `Serial Terminal from nRF Connect for Desktop`_.
+
+      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+
+      We recommend adding the nRF Util path to your environmental variables.
+
+.. _nrf54h20_install_ncs:
+
+Installing the |NCS|
+====================
+
+After you have installed the toolchain, complete the following steps to get the customer sampling version of the |NCS|:
+
+1. In the terminal window opened during the toolchain installation, initialize west with the revision of the |NCS| from the customer sampling by running the following command:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.6.99-cs2
+
+#. Enter the following command to clone the project repositories::
+
+      west update
+
+   Depending on your connection, this might take some time.
+
+#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+
+      west zephyr-export
+
+Your directory structure now looks similar to this::
+
+    ncs-lcs/work-dir
+    |___ .west
+    |___ bootloader
+    |___ modules
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+    |___ ...
+
+Note that there are additional folders, and that the structure might change.
+The full set of repositories and folders is defined in the manifest file.
+
+Installing the Terminal application
+***********************************
 
 On your computer, install `nRF Connect for Desktop`_.
 You must also install a terminal emulator, such as `nRF Connect Serial Terminal`_ (from the nRF Connect for Desktop application) or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).
```

Comment:
This could probably be moved Required software, where git and curl are already mentioned.

Reply:
I'd disagree on suggesting this before the toolchain installation, as the nrfconnect for vscode will want to know ncs path, and nrf connect for desktop can also install the toolchain, but the wrong one. Till we drop the bootstrap script this should be done after.

Reply:
Ah, that makes sense for this.